asteriskserver/scripts/update_audio_assets.rb

```diff
@@ -48,26 +53,25 @@
   else
     info = Sndfile::File.info(src)    
     out_ext = sln_ext[info.samplerate]
+    is_raw = true
     raise "#{src} has an unsuppored sampling rate of #{info.samplerate}" unless out_ext
   end
 
   dst = base + "." + out_ext
   #ditch if the file exists and the dest is newer
-  next if File.exist?(dst) and File.mtime(dst) > File.mtime(src)
+  next if File.exist?(dst) and File.mtime(dst) > File.mtime(src) and File.mtime(dst) > script_mtime
 
   #make the dir
   FileUtils.mkdir_p(File.dirname(dst))
 
   #report
   puts "#{src} -> #{dst}"
 
-  #convert gsm to wav so we can normalize
-  if src_ext == ".gsm"
-    wav = base + "-tmp.wav"
-    raise "cannot create tmp wav from gsm" unless system("sndfile-convert", "-pcm16", src, wav)
-    cleanup << wav
-    src = wav
-  end
+  #convert to wav
```

Comment:
is the other comment more accurate? we are converting gsm to wav so we can later work with it? Or does this command do anything like normalization?